site/docs/v1/tech/admin-guide/troubleshooting.adoc

```diff
@@ -302,10 +302,30 @@ To fix your data, follow these steps:
 * Find the user's Id.
 * Modify the problematic `data` fields via the User API to remove the mismatched fields.
 
-Unfortunately, you can't search for the user by email, username, or other characteristics using the User Search API. If you have the user's Id, which may be available in the error messages, you can use that. Searching by UUID doesn't use Elasticsearch. If you do not have the UUID, you should change to the database search engine to find the user. You can switch back to Elasticsearch after you have corrected the `data` field.
+Unfortunately, you often can't search for the user by email, username, or other characteristics using the User Search API. If you have the user's Id, which may be available in the error messages, you can use that. Searching by UUID doesn't use Elasticsearch. You can also try to search for all emails, using a `queryString` of `email:*` if you don't have too many users. Another option is to change to the database search engine to find the user. You can switch back to Elasticsearch after you have corrected the `data` field.
 
 If you have a support contract, please https://account.fusionauth.io/account/support/[open a ticket] and we'll help you out. If not, please back up your FusionAuth database and test the fix thoroughly in a staging environment before applying it to your production instance.
 
+Suppose you had a data field called `customPermissions` which was a `string` on some users and an `object` on others. Below is a script which converts the type of `customPermissions` from `string` to an `object` for all users.
```

Comment:
Maybe we could disclaim that this type of conversion is straightforward and aided by the use of the parse function.  A more complex data mismatch would be a heavier code lift (theoretically).

```
JSON.parse(user.data.customPermissions)
```

Reply:
Good point. Added some color.